src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import ListPropType from '../../prop-types/list';
+import './Tooltip.scss';
+
+const propTypes = {
+    children: PropTypes.element.isRequired,
+    tooltip: PropTypes.element.isRequired,
+    position: ListPropType(['top-left', 'top-right', 'bottom-right', 'bottom-left']),
```

Comment:
do we support / need horizontally centred tooltip? I think it's the behaviour of the previous tooltip

Reply:
Good point, I'll check with Antoine 👍

Reply:
we do need a centred tooltip, I'll add this...